src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -201,8 +201,15 @@ class Manager : public DoutPrefixProvider {
     if ( (topic_persistency_ttl != 0 && event_entry.creation_time != ceph::coarse_real_clock::zero() &&
          time_now - event_entry.creation_time > std::chrono::seconds(topic_persistency_ttl))
          || ( topic_persistency_max_retries != 0 && entry_persistency_tracker.retires_num >  topic_persistency_max_retries) ) {
-      ldpp_dout(this, 20) << "Expiring entry retry_number=" << entry_persistency_tracker.retires_num << " creation_time="
-                          << event_entry.creation_time << " time_now:" << time_now << dendl;
+      ldpp_dout(this, 1) << "Expiring entry for topic= "
```

Comment:
changing loglevel from 20 to 1 for this one may not right, a part from that change looks good

Reply:
Its purposely changed to log level 1, so we can monitor the logs for messages that are expired.
unfortunately with log level 20, its super verbose and cannot be turned on all the time to monitor the expired notification.
Also expired notification is as good as errored notification, so to have some visibility need to make the log level 1.
this PR is inline with similar PR for sync notification (non-persistent notification) https://github.com/ceph/ceph/pull/52876/files

Reply:
> changing loglevel from 20 to 1 for this one may not right, a part from that change looks good

Its purposely changed to log level 1, so we can monitor the logs for messages that are expired.
unfortunately with log level 20, its super verbose and cannot be turned on all the time to monitor the expired notification.
Also expired notification is as good as errored notification, so to have some visibility need to make the log level 1.
this PR is inline with similar PR for sync notification (non-persistent notification) https://github.com/ceph/ceph/pull/52876/files

Reply:
Agree that this should be level 1.
Entry expiry may cause information loss, so shoudl be considered an error case.